cmake/external/openblas.cmake

```diff
@@ -22,7 +22,7 @@ IF(NOT WIN32)
         CACHE FILEPATH "openblas library." FORCE)
     SET(CBLAS_INC_DIR "${CBLAS_INSTALL_DIR}/include" CACHE PATH "openblas include directory." FORCE)
     SET(OPENBLAS_CC "${CMAKE_C_COMPILER} -Wno-unused-but-set-variable -Wno-unused-variable")
-    SET(OPENBLAS_COMMIT "v0.2.20")
+    SET(OPENBLAS_COMMIT "v0.3.7")
```

Comment:
Why upgrade the openblas?

Reply:
Errors will be reported during compilation when using the v0.2.20. Update to the latest just be fine.

Will this change have any other impact？

Reply:
I don't know whether this change will have any other impact.
Could you paste the errors here? Or how about upgrade the openblas in a new PR?

Reply:
emm.. But the need (update openblas) comes from Jetson's support. Without this update in this pr, paddle can't be compiled on jetson. 

The error log about the openblas v0.2.20 can seen above.

Reply:
I mean you can merge the independent OpenBlas PR at first, and see if there is some error in CE platform?

Reply:
ok